docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -0,0 +1,575 @@
+        performance improvements.
+      - ``true``
+
+The following table describes performance tuning catalog properties for the
+connector.
+
+.. warning::
+
+   Performance tuning configuration properties are considered expert-level
+   features. Altering these properties from their default values is likely to
+   cause instability and performance degradation. We strongly suggest that
+   you use them only to address non-trivial performance issues, and that you
+   keep a backup of the original values if you change them.
+
+.. list-table:: Delta Lake performance tuning configuration properties
+    :widths: 30, 50, 20
+    :header-rows: 1
+
+    * - Property name
+      - Description
+      - Default
+    * - ``delta.domain-compaction-threshold``
+      - Sets the number of transactions to act as threshold. Once reached the
+        connector initiates compaction of the underlying files and the delta
+        files. A higher compaction threshold means reading less data from the
+        underlying data source, but a higher memory and network consumption.
+      - 100
+    * - ``delta.max-outstanding-splits``
+      - The target number of buffered splits for each table scan in a query,
+        before the scheduler tries to pause.
+      - 1000
+    * - ``delta.max-splits-per-second``
+      - Sets the maximum number of splits used per second to access underlying
+        storage. Reduce this number if your limit is routinely exceeded, based
+        on your filesystem limits. This is set to the absolute maximum value,
+        which results in Trino maximizing the parallelization of data access
+        by default. Attempting to set it higher results in Trino not being
+        able to start.
+      - Integer.MAX_VALUE
+    * - ``delta.max-initial-splits``
+      - For each query, the coordinator assigns file sections to read first
+        at the ``initial-split-size`` until the ``max-initial-splits`` is
+        reached. Then, it starts issuing reads of the ``max-split-size`` size.
+      - 200
+    * - ``delta.max-initial-split-size``
+      - Sets the initial :ref:`prop-type-data-size` for a single read section
+        assigned to a worker until ``max-initial-splits`` have been processed.
+        You can also use the corresponding catalog session property
+        ``<catalog-name>.max_initial_split_size``.
+      - ``32MB``
+    * - ``delta.max-split-size``
+      - Sets the largest :ref:`prop-type-data-size` for a single read section
+        assigned to a worker after max-initial-splits have been processed. You
+        can also use the corresponding catalog session property
+        ``<catalog-name>.max_split_size``.
+      - ``64MB``
+
+The following table describes :ref:`catalog session properties
+<session-properties-definition>` supported by the Delta Lake connector to
+configure processing of Parquet files.
+
+.. list-table:: Parquet catalog session properties
```

Comment:
In Delta, are files are Parquet files, so this distinction isn't desireable.

cc @findinpath @krvikash

Reply:
> In Delta, all files are Parquet files, so this distinction isn't desirable.

Can you please add a bit more detail to your comment/suggestion?

Reply:
for example 
"Parquet catalog session properties" -> "catalog session properties"

remove "configure processing of Parquet files."

Reply:
This PR has been merged for over a year .. please discuss this all on a new PR.

Reply:
I'll create a follow-up PR with all delta session properties.

Reply:
> This PR has been merged for over a year 

@mosabua is there some specific amount of time after which  feedback is no longer welcome?

@findinpath thank you

Reply:
No there is no specific time @findepi .. commenting on an old PR just very much limits the audience of who will find out about an issue. The docs might have wildly changed since the PR was merged as well.